docs/development/security/risk_assesment.md

```diff
@@ -0,0 +1,15 @@
+# Risk assesment
+From [issue 171](https://github.com/aau-giraf/wiki/issues/171).
+
+The risk assesment can be found here: [PDF](risk_assesment.pdf)
+
+The risk assesment have been based on the guidance of Datatilsynet.
```

Comment:
```suggestion
The risk assessment has been based on the guidance of Datatilsynet.
```

Reply:
I believe i have fixed the spelling mistakes and i also corrected the file names.